constantinCurtis/src/main/java/data/collection/task/Interfaces/Order.java

```diff
@@ -0,0 +1,12 @@
+package data.collection.task.Interfaces;
+
```

Comment:
According to Java code convention packages should always be written in all-lowercase.
See https://www.oracle.com/technetwork/java/codeconventions-135099.html

Reply:
Yes I know about that, but It was created by habit, my mistake. Thank you, changed